LICENSE.md

```diff
@@ -0,0 +1,6 @@
+Copyright (c) 2011, The Multiverse Team All rights reserved.
```

Comment:
Should probably change this to match the current year, both here and in the README.

Reply:
Agreed.

Reply:
Actually, probably remove the license from the README.

Reply:
Then readme will be basically empty haha, I think time for #2519 as well